.golangci.example.yml

```diff
@@ -449,6 +449,12 @@ linters-settings:
     servingv1: knative.dev/serving/pkg/apis/serving/v1
     # using `autoscalingv1alpha1` alias for `knative.dev/serving/pkg/apis/autoscaling/v1alpha1` package
     autoscalingv1alpha1: knative.dev/serving/pkg/apis/autoscaling/v1alpha1
+  gomodreplace:
+    # Allow local `replace` directives. Default is false.
+    local: false
```

Comment:
I am failing to think of a use case where I want to disallow all replace except for local. (please correct me if my understanding of how this works is wrong.) iirc we discussed the following use cases:
- disallow only local replace
- disallow all replace
- disallow all replace except for some modules

Reply:
a use case: https://github.com/hashicorp/consul/blob/master/go.mod

Reply:
can you elaborate on how it is used or why it is required?

if this is a library it means that tests run from within and without would behave differently, something that seems bad. if this is a main module, it still seems gross to maintain, and I think using the `allow-list` syntax is cleaner.

Reply:
I don't know exactly why they are using that, I saw that pattern several times.

I think it is a way of facilitating the development with the multi modules

Reply:
we can add it if people really want it, but that sounds like a things for local development, not something that should be merged or released